app/src/main/java/com/android/streetworkapp/ui/park/AddEvent.kt

```diff
@@ -0,0 +1,305 @@
+            navigationIcon = {
+              IconButton(
+                  onClick = { navigationActions.goBack() },
+                  modifier = Modifier.testTag("goBackButton")) {
+                    Icon(
+                        Icons.AutoMirrored.Filled.ArrowBack, contentDescription = "Arrow Back Icon")
+                  }
+            })
+      }) { innerPadding ->
+        Box(modifier = Modifier.padding(innerPadding).fillMaxSize().testTag("addEventScreen")) {
+          Column(
+              modifier = Modifier.fillMaxWidth(),
+              verticalArrangement = Arrangement.spacedBy(18.dp),
+              horizontalAlignment = Alignment.CenterHorizontally) {
+                Text(
+                    text = "Event Creation",
+                    fontSize = 24.sp,
+                )
+                EventTitleSelection(event)
+                EventDescriptionSelection(event)
+                TimeSelection(event)
+                Text(
+                    text = "How many participants do you want?",
+                    fontSize = 18.sp,
+                )
+                ParticipantNumberSelection(event)
+              }
+          FloatingActionButton(
+              onClick = {
+                // TODO: use one of the future viewModel to add the event to firebase
+                navigationActions.goBack()
+              },
+              modifier =
+                  Modifier.align(Alignment.BottomCenter)
+                      .padding(40.dp)
+                      .size(width = 150.dp, height = 40.dp)
+                      .testTag("addEventButton"),
+          ) {
+            Text("Add new event")
+          }
+        }
+      }
+}
+
+/**
+ * Used to change the title of the event
+ *
+ * @param event the event that will be updated
+ */
+@Composable
+fun EventTitleSelection(event: Event) {
+  var title by remember { mutableStateOf("") }
+
+  OutlinedTextField(
+      value = title,
+      onValueChange = {
+        title = it
+        event.title = title
+      },
+      label = { Text("What kind of event do you want to create?") },
+      modifier = Modifier.testTag("titleTag").fillMaxWidth(0.9f).height(64.dp))
+}
+
+/**
+ * Used to change the description of the event
+ *
+ * @param event the event that will be updated
+ */
+@Composable
+fun EventDescriptionSelection(event: Event) {
+  var description by remember { mutableStateOf("") }
+
+  OutlinedTextField(
+      value = description,
+      onValueChange = {
+        description = it
+        event.description = description
+      },
+      label = { Text("Describe your event:") },
+      modifier = Modifier.testTag("descriptionTag").fillMaxWidth(0.9f).height(128.dp))
+}
+
+/**
+ * Used to change the maximum number of participants of the event
+ *
+ * @param event the event that will be updated
+ */
+@Composable
+fun ParticipantNumberSelection(event: Event) {
+  val minParticipants = 2 // We need at least 2 users for every event
```

Comment:
Improvement: These 2 shouldn't be here, they should be put in another file so that we can easily access then inside other components. (I think these values are already hardcoded in some other tests that we should change but that's beside the scope of this PR)
 They should also be renamed to MIN_PARTICIPANTS AND MAX_PARTICIPANTS to make it clear that these values are constants.

Reply:
You are right, I defined with an Object "EventConstants" in the Event.kt file. Thank you!